### ISSUE TITLE
Issue 421 - Step 5 - Enable a PM selection for an Admin user

### ISSUE THREAD

**Issue by FWAJL** [FWAJL/FieldWorkAssistantMVC#749]
In order to set it as current a selected PM and view the data for that PM.

**Comment by damirius** [comment]
@bsaiken

You can use right click context menu on user/listAll page when you login as admin. Just right click any user and click 'switch'. It will take you back to the login page, fill out the info and submit it for you.
You can check it out on feature_damir branch.
Thanks
@FWAJL 

I opted for redirecting the user back to the login and filling it up automatically instead of making ajax call to /auth from user/listAll page. The reason for this is that login runs `disconnect`/`logout` method first which destroys the session. The problem is that if I run that inside the auth method instead it will not work, since some stuff is stored in session on the controller call which we need.

**Comment by FWAJL** [comment]
@damirius @bsaiken 

I think we could just show the left menu of the PMs without logging out / logging in. 
The list of PM Session array should be stored in the PmsSessionArray (or similar, can't remember the exact spelling).
Logging in as a PM is basically adding one single PM session array to this array of PM session array.

As an admin, if we set the Current PM Session to be the one selected, then the admin can do whatever the PM can.
I looked at this a while back and I think it is possible.

However, if this works then it is fine for now. We'll see soon enough.

Thank you again for your work.
